datasets/drenthe/time_curves/energy_distribution_woody_biomass_time_curve.csv

```diff
@@ -0,0 +1,83 @@
+year,max_demand
```

Comment:
These time curve files don't look right to me. It's probably an issue with the scaling script and not your fault; I will look into it in the morning.